robocop/checkers/misc.py

```diff
@@ -577,6 +577,36 @@
         """,
         added_in_version="4.0.0",
     ),
+    "0926": Rule(
```

Comment:
BTW Have you seen many cases like this? I am feeling that this rule is a bit redundant, because it checks for a very specific typo. I know that it may result in a wrong condition, but double negation may be put nearly everywhere where Python evaluation is performed, so why we look for it only here?

Reply:
I saw, but only once in my life :) I have also wondered after raising PR if it's good candidate for rule. Quite specific like you said. I could park this branch for time being and revisit it in the future to decide if it's still needed

Reply:
Agreed. It may also become a community rule, though it's universal (but a very rare case).